.travis.yml

```diff
@@ -36,11 +36,11 @@ before_install:
 
 env:
   global:
-  - LINT_CHECK=0 TESTS="0" VERSION="saas-17" ODOO_REPO="odoo/odoo"
+  - LINT_CHECK=0 TESTS="0" VERSION="saas-17-valuation-merge" ODOO_REPO="odoo-dev/odoo"
```

Comment:
The line 39 should be the next
`- LINT_CHECK=0 TESTS="0" VERSION="saas-17" ODOO_BRANCH="saas-17-valuation-merge" ODOO_REPO="odoo-dev/odoo"`